.github/workflows/trigger-update-extensions-galleries-pipeline.yml

```diff
@@ -0,0 +1,21 @@
+name: Trigger Update Galleries Pipeline
+
+on:
+  push:
+    branches:
```

Comment:
You should add the file filter as well so it only runs when the gallery files themselves are updated : https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#using-filters-to-target-specific-paths-for-pull-request-or-push-events

Reply:
added!